pkg/apis/push/v1alpha1/unifiedpushserver_types.go

```diff
@@ -4,15 +4,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 // UnifiedPushServerSpec defines the desired state of UnifiedPushServer
 // +k8s:openapi-gen=true
 type UnifiedPushServerSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
```

Comment:
It is missing the CR Status. How the admin/user can check the data used in the case of failures?
Following an example over the impl here : https://github.com/aerogear/mobile-security-service-operator/pull/125 (Note that  added XXX in the fields
<img width="599" alt="Screenshot 2019-06-28 at 09 27 28" src="https://user-images.githubusercontent.com/7708031/60342152-11379200-9987-11e9-976b-d646b56756de.png">
 just to not expose them here)

Reply:
They could just look in the secrets that they've specified in the spec section.

Reply:
@grdryn if something goes wrong and it is not using what was spec then you will not able to check it. Also, see that many of data should be generated dynamically. The cronJobStatus also its important.

Reply:
We can do it after too if you wish.